content/en/docs/setup/production-environment/tools/kops.md

```diff
@@ -232,5 +232,5 @@ including tools for logging, monitoring, network policy, visualization, and cont
 * Learn more about `kOps` [advanced usage](https://kops.sigs.k8s.io/) for tutorials,
   best practices and advanced configuration options.
 * Follow `kOps` community discussions on Slack:
-  [community discussions](https://github.com/kubernetes/kops#other-ways-to-communicate-with-the-contributors).
+  [community discussions](https://kubernetes.slack.com/archives/C3QUFP0QM).
```

Comment:
Please add:
```markdown
 (visit https://slack.k8s.io/ for an invitation to this Slack workspace).
```
as otherwise people might not know how to join.

Reply:
@sftim Thanks for the suggestion. Changes made as per suggestion